### DISCUSSION TITLE
how does maintain "count"?

### DISCUSSION THREAD

**Discussion by MaskDuck** [lowlighter/metrics#976] [Q&A] [answered]
question, maintaining counts only repositories with maintain+ perm? 
right now i have write access to a project with 1k+ stars but it doesn't show.

**Comment by lowlighter** [comment, marked as answer]
Hi 👋 !

If you're talking about the stats located under the two-weeks calendar, it uses the following from GitHub GraphQL API:
https://github.com/lowlighter/metrics/blob/de323b18fbbfbae04f1d5004ae5ba011367ee3dc/source/plugins/base/queries/user.x.graphql#L30-L32

Else if you were asking about the repositories sections, then it depends to the value of [`repositories_affiliations`](https://github.com/lowlighter/metrics/blob/master/source/plugins/base/README.md#repositories_affiliations), which is set to the `owner` constraints by default. You can extend it to `collaborator`, `organization_member` or the broadest one which is no constraints: `""`

Additionally, [`repositories_forks`](https://github.com/lowlighter/metrics/blob/master/source/plugins/base/README.md#repositories_forks) are also ignored by default in case you were wondering, but can be opted-in too

**Reply by MaskDuck** [reply]
thanks!
